doc-content/enterprise-only/bpmn/bpmn-intermediate-events-ref.adoc

```diff
@@ -79,7 +79,7 @@ A message intermediate event is an intermediate event that enables you to manage
 
 [[_timer_intermediate_event]]
 .Timer
-A timer intermediate event enables you to delay workflow execution or to trigger the workflow execution periodically. It represents a timer that can trigger one or multiple times after a specified period of time. When the timer intermediate event is triggered, the timer condition (the defined time) is checked and the outgoing flow is taken. 
+A timer intermediate event enables you to delay workflow execution or to trigger the workflow execution periodically. It represents a timer that can trigger one or multiple times after a specified period of time. When the timer intermediate event is triggered, the timer condition (the defined time) is checked and the outgoing flow is taken.
```

Comment:
Consider rewording "the timer condition (the defined time) is checked and the outgoing flow is taken." 

to

the timer condition, which is the defined time,, is checked and the outgoing flow is used.

Reply:
I will address this in the next update because it requires QE review.